makefiles/kconfig.mk

```diff
@@ -88,11 +98,17 @@ USEPKG_W_PREFIX = $(addprefix PKG_,$(USEPKG))
 # defining symbols like 'MODULE_<MODULE_NAME>' or PKG_<PACKAGE_NAME> which
 # default to 'y'. Then, every module and package Kconfig menu will depend on
 # that symbol being set to show its options.
+# Do nothing when testing Kconfig module dependency modelling.
```

Comment:
this is not just for `testing`, but for actually `performing` the dependency resolution?

Reply:
Well, the `TEST_KCONFIG` variable is there only during the transition period and it's meant for testing. How would you reword this?